frontend/globals/unity-util.ts

```diff
@@ -673,6 +695,104 @@ export class UnityUtil {
 		UnityUtil.toUnity('StopMeasuringTool', UnityUtil.LoadingState.MODEL_LOADING, undefined);
 	}
 
+	/**
+	 * Enable the measure tool toolbar.
+	 * @category Measuring tool
+	 */
+	public static enableMeasureToolToolbar() {
+		UnityUtil.toUnity('EnableMeasureToolToolbar', undefined, undefined);
+	}
+
+	/**
+	 * Enable the measure tool toolbar.
+	 * @category Measuring tool
+	 */
+	public static disableMeasureToolToolbar() {
+		UnityUtil.toUnity('DisableMeasureToolToolbar', undefined, undefined);
+	}
+
+	/**
+	 * Set the measure tool mode.
+	 * @category Measuring tool
+	 * @param mode - The measuring mode, accepted values are "Point", "Raycast", "MinimumDistance",
+	 * "SurfaceArea" or "PolygonArea".
+	 */
+	public static setMeasureToolMode(mode) {
+		UnityUtil.toUnity('SetMeasureToolMode', undefined, mode);
+	}
+
+	/**
+	 * Set the measure tool units.
+	 * @category Measuring tool
+	 * @param units - The measuring units accepted values are "cm", "mm", "m"
+	 */	public static setMeasureToolUnits(units) {
+		UnityUtil.toUnity('SetMeasureToolUnits', undefined, units);
+	}
+
+	/**
+	 * Enable the measure tool snap to edges.
+	 * @category Measuring tool
+	 */
+	public static enableMeasureToolSnap() {
+		UnityUtil.toUnity('EnableMeasureToolSnap', undefined, undefined);
+	}
+
+	/**
+	 * Disable the measure tool snap to edges.
+	 * @category Measuring tool
+	 */
+	public static disableMeasureToolSnap() {
+		UnityUtil.toUnity('DisableMeasureToolSnap', undefined, undefined);
+	}
+
+	/**
+	 * Clear all measurements
+	 * @category Measuring tool
+	 */
+	public static clearMeasureToolMeasurements() {
```

Comment:
Can we change this to clearAllMeasurements or something similar? Let's avoid similarly named functions with a difference of a plural. it gets hard to read 😆